articles/how-artificially-intelligent-chatbots-have-impacted-healthcare-industry/index.md

```diff
@@ -0,0 +1,59 @@
+### Introduction
+In recent times, organizations have adopted the benefits offered by AI-powered [virtual assistants](https://searchcustomerexperience.techtarget.com/definition/virtual-assistant-AI-assistant) to automate routine tasks and give their clients attention in customer services, payments, and marketing.
+
+The healthcare industry has now adopted this trend. [AI](https://www.ibm.com/cloud/learn/what-is-artificial-intelligence) has created an environment where the patient outcome has been improved with minimum costs involved.
+
+AI technology is not new to healthcare. The urge for cost reduction and improve patient outcomes and data availability have led to various [innovations](https://innolytics-innovation.com/what-is-innovation/) in the industry, ranging from early disease diagnosis to the introduction of life-saving medicines.
```

Comment:
The motive to reduce costs, improve patient recoveries, and enhance data availability has led to many innovations in the industry. These range from early disease diagnosis to the introduction of life-saving medicine. 

Observe the tense changes made. Maintain the same tense structure and tonality. Also ensure sentences are small and not compounded using and, commas.